### DISCUSSION TITLE
RITA documentation is outdated but it still works.

### DISCUSSION THREAD

**Discussion by Mastadamus** [Security-Onion-Solutions/securityonion#3973] [General] [answered]
RITA can still be installed with Sec Onion 2.3 but the documentation is a bit outdated. All the commands are generally right but "bro" needs to be replaced with "zeek" for every command.  The documentation is here. https://docs.securityonion.net/en/2.3/rita.html

**Comment by dougburks** [comment, marked as answer]
Updated:
https://github.com/Security-Onion-Solutions/securityonion-docs/commit/4492a5958c2275d45a1df7e207e3a07ba360ec57#diff-547a1e4d2b2de6452f26f85e9d363e79bf332dfdb1648baf8e1b28e9e103aa10

**Reply by Mastadamus** [reply]
Thank you. That was very quick.
